debian/rules

```diff
@@ -13,9 +13,9 @@ endif
 
 DEBVERS := $(shell dpkg-parsechangelog | sed -n -e 's/^Version: //p')
 VERSION := $(shell echo '$(DEBVERS)' | sed -e 's/^[\[:digit:]]*://' -e 's/[-].*//')
-MAJOR := $(shell echo '$(VERSION)' | awk -F . {'print $$1'})
-MINOR := $(shell echo '$(VERSION)' | awk -F . {'print $$2'})
-REV := $(shell echo '$(VERSION)' | awk -F . {'print $$3'})
+MAJOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$1'})
+MINOR := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$2'})
+REV := $(shell echo '$(VERSION)' | awk -F '[.+]' {'print $$3'})
```

Comment:
ah I see; so I guess I should use the same format as previously: 1.4.79. Thx I'll update this.